values.yaml

```diff
@@ -53,3 +53,49 @@ reconcile:
   overwriteZk: true
   successfulJobsHistoryLimit: null
   failedJobsHistoryLimit: null
+
+service:
+  type: ClusterIP
```

Comment:
Changing type would make sense only to make this service a `NodePort`. But using NodePort usually implies possibility to configure static port for binding. So, currently chaging service type does not make any benefit and I'd like to keep it hardcoded to `ClusterIP`